draft-ietf-tls-tls13.md

```diff
@@ -3567,30 +3577,64 @@ and their allocation policies (unchanged from {{RFC4346}}) are listed below.
 
 This document also uses a registry originally created in {{RFC4366}}. IANA has
 updated it to reference this document. The registry and its allocation policy
-(unchanged from {{RFC4366}}) is listed below:
-
--  TLS ExtensionType Registry: Future values are allocated via IETF
-  Consensus {{RFC2434}}.  IANA [shall update/has updated] this registry
-  to include the "key_share", "pre_shared_key", and "early_data"
-  extensions as defined in this document. IANA [shall update/has updated]
-  this registry to include an "TLS 1.3" column with the following three
-  values: "Clear", indicating that they shall be in the ServerHello
-  "Encrypted", indicating that they shall be in the EncryptedExtensions
-  block, and "Forbidden" indicating that they are not used in TLS 1.3.
-  This registry [shall be/has been] initially populated with the values in this
-  document.
-
-This document also uses two registries originally created in {{RFC4492}}. IANA
-[should update/has updated] it to reference this document. The registries
-and their allocation policies are listed below.
-
-- TLS NamedCurve registry: Future values are allocated via IETF Consensus
-  {{RFC2434}}.
-
-- TLS ECPointFormat Registry: Future values are allocated via IETF Consensus
-  {{RFC2434}}.
-
-In addition, this document defines two new registries to be maintained by IANA:
+is listed below:
+
+- TLS ExtensionType Registry: Values with the first byte in the range
+   0-254 (decimal) are assigned via Specification Required {{RFC2434}}.
+   Values with the first byte 255 (decimal) are reserved for Private
+   Use {{RFC2434}}. IANA [SHALL update/has updated]
+   this registry to include the "key_share", "pre_shared_key", and
+   "early_data" extensions as defined in this document.
+
+   IANA [shall update/has updated] this registry to include a "TLS
+   1.3" column with the following four values: "Client", indicating
+   that the server shall not send them. "Clear", indicating
+   that they shall be in the ServerHello "Encrypted", indicating that
```

Comment:
missing period after ServerHello

Even with that fixed, it's a little cumbersome to read. Can you do a bulleted list instead?